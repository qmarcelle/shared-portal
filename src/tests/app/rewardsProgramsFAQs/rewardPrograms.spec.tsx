import RewardProgramsFaqs from '@/app/myHealth/rewardsProgramsFAQs/page';
import '@testing-library/jest-dom';
import { fireEvent, render, screen } from '@testing-library/react';

const renderUI = () => {
  return render(<RewardProgramsFaqs />);
};

describe('PharmacyDrugInformation', () => {
  it('should render UI correctly', async () => {
    const component = renderUI();

    screen.getByRole('heading', { name: 'Rewards Program FAQs' });
    screen.getByText(
      'If you need help, call [1-800-000-000] or email help@bcbstrewards.com.',
    );
    screen.getByText('About The Rewards Programs');
    screen.queryAllByText('50 Points');
    screen.queryByText('Personal Health Assessment (PHA)');
    screen.queryByText(
      'Take your assessment first to start earning rewards points',
    );

    screen.getByText('How much can my family earn?');
    //check that accordion is not open at first
    let informationBody = screen.queryByText(
      'You and your spouse (if covered) can earn up to $100 each quarter, and a maximum of $400 for each of you per calendar year.',
    );
    expect(informationBody).not.toBeInTheDocument();

    //check that accordion opens when clicked
    fireEvent.click(screen.getByText('How much can my family earn?'));
    informationBody = screen.getByText(
      'You and your spouse (if covered) can earn up to $100 each quarter, and a maximum of $400 for each of you per calendar year.',
    );
    expect(informationBody).toBeInTheDocument();

    //check that it closes when clicked again
    fireEvent.click(screen.getByText('How much can my family earn?'));
    informationBody = screen.queryByText(
      'You and your spouse (if covered) can earn up to $100 each quarter, and a maximum of $400 for each of you per calendar year.',
    );
    expect(informationBody).not.toBeInTheDocument();

    expect(component.baseElement).toMatchSnapshot();
  });
});
