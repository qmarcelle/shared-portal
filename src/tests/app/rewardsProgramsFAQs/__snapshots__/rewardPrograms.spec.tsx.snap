// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PharmacyDrugInformation should render UI correctly 1`] = `
<body>
  <div>
    <div
      class="flex flex-col justify-center items-center page"
    >
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <div
        class="flex flex-col app-content app-base-font-color"
      >
        <h1
          class="title-1 mb-0"
        >
          Rewards Program FAQs
        </h1>
        <span
          style="display: block; width: 1px; min-width: 1px; height: 16px; min-height: 16px;"
        />
        <p
          class="body-1 body-1 mb-2"
          style="display: block; max-width: 100%;"
        >
          If you need help, call [1-800-000-000] or email help@bcbstrewards.com.
        </p>
        <section
          class="flex flex-row items-start app-body"
        >
          <div
            class="flex flex-col flex-grow basis-6/12 items-stretch mt-4"
          >
            <div
              class="card-main undefined"
            >
              <div
                class="flex flex-col m-4"
              >
                <h2
                  class="title-2 ml-2 mt-4"
                >
                  How to Earn Reward Points
                </h2>
                <span
                  style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
                />
                <div
                  class="card-button cursor-pointer mb-4"
                >
                  <section
                    class="flex flex-row"
                  >
                    <div
                      class="flex flex-col p-4 h-auto w-[72px] items-center text-center justify-center bg-primary-focus rounded-l-[8px]"
                    >
                      <p
                        class="body-1 text-white"
                        style="display: block; max-width: 100%;"
                      >
                        50 Points
                      </p>
                    </div>
                    <div
                      class="flex flex-col px-5 py-4"
                    >
                      <p
                        class="body-1 body-1 font-bold my-2"
                        style="display: block; max-width: 100%;"
                      >
                        Personal Health Assessment (PHA)
                      </p>
                      <p
                        class="body-1"
                        style="display: block; max-width: 100%;"
                      >
                        Take your assessment first to start earning rewards points
                      </p>
                    </div>
                  </section>
                </div>
                <div
                  class="card-button cursor-pointer mb-4"
                >
                  <section
                    class="flex flex-row"
                  >
                    <div
                      class="flex flex-col p-4 h-auto w-[72px] items-center text-center justify-center bg-primary-focus rounded-l-[8px]"
                    >
                      <p
                        class="body-1 text-white"
                        style="display: block; max-width: 100%;"
                      >
                        100 Points
                      </p>
                    </div>
                    <div
                      class="flex flex-col px-5 py-4"
                    >
                      <p
                        class="body-1 body-1 font-bold my-2"
                        style="display: block; max-width: 100%;"
                      >
                        Annual Preventive Exam
                      </p>
                      <p
                        class="body-1"
                        style="display: block; max-width: 100%;"
                      >
                        1 visit per year max
                      </p>
                    </div>
                  </section>
                </div>
                <div
                  class="card-button cursor-pointer mb-4"
                >
                  <section
                    class="flex flex-row"
                  >
                    <div
                      class="flex flex-col p-4 h-auto w-[72px] items-center text-center justify-center bg-primary-focus rounded-l-[8px]"
                    >
                      <p
                        class="body-1 text-white"
                        style="display: block; max-width: 100%;"
                      >
                        50 Points
                      </p>
                    </div>
                    <div
                      class="flex flex-col px-5 py-4"
                    >
                      <p
                        class="body-1 body-1 font-bold my-2"
                        style="display: block; max-width: 100%;"
                      >
                        Register for Teladocâ„¢ Health
                      </p>
                      <p
                        class="body-1"
                        style="display: block; max-width: 100%;"
                      >
                        New registration only | 1 registration per person
                      </p>
                    </div>
                  </section>
                </div>
                <div
                  class="card-button cursor-pointer mb-4"
                >
                  <section
                    class="flex flex-row"
                  >
                    <div
                      class="flex flex-col p-4 h-auto w-[72px] items-center text-center justify-center bg-primary-focus rounded-l-[8px]"
                    >
                      <p
                        class="body-1 text-white"
                        style="display: block; max-width: 100%;"
                      >
                        50 Points
                      </p>
                    </div>
                    <div
                      class="flex flex-col px-5 py-4"
                    >
                      <p
                        class="body-1 body-1 font-bold my-2"
                        style="display: block; max-width: 100%;"
                      >
                        Teladoc Health Use
                      </p>
                      <p
                        class="body-1"
                        style="display: block; max-width: 100%;"
                      >
                        Per each use | 100 points max per year
                      </p>
                    </div>
                  </section>
                </div>
                <div
                  class="card-button cursor-pointer mb-4"
                >
                  <section
                    class="flex flex-row"
                  >
                    <div
                      class="flex flex-col p-4 h-auto w-[72px] items-center text-center justify-center bg-primary-focus rounded-l-[8px]"
                    >
                      <p
                        class="body-1 text-white"
                        style="display: block; max-width: 100%;"
                      >
                        1 Point
                      </p>
                    </div>
                    <div
                      class="flex flex-col px-5 py-4"
                    >
                      <p
                        class="body-1 body-1 font-bold my-2"
                        style="display: block; max-width: 100%;"
                      >
                        Daily Steps
                      </p>
                      <p
                        class="body-1"
                        style="display: block; max-width: 100%;"
                      >
                        Per 5,000 steps | 100 points max per year
                      </p>
                    </div>
                  </section>
                </div>
                <div
                  class="card-button cursor-pointer mb-4"
                >
                  <section
                    class="flex flex-row"
                  >
                    <div
                      class="flex flex-col p-4 h-auto w-[72px] items-center text-center justify-center bg-primary-focus rounded-l-[8px]"
                    >
                      <p
                        class="body-1 text-white"
                        style="display: block; max-width: 100%;"
                      >
                        50 Points
                      </p>
                    </div>
                    <div
                      class="flex flex-col px-5 py-4"
                    >
                      <p
                        class="body-1 body-1 font-bold my-2"
                        style="display: block; max-width: 100%;"
                      >
                        Flu Shot
                      </p>
                      <p
                        class="body-1"
                        style="display: block; max-width: 100%;"
                      >
                        1 per year max
                      </p>
                    </div>
                  </section>
                </div>
                <div
                  class="card-button cursor-pointer mb-4"
                >
                  <section
                    class="flex flex-row"
                  >
                    <div
                      class="flex flex-col p-4 h-auto w-[72px] items-center text-center justify-center bg-primary-focus rounded-l-[8px]"
                    >
                      <p
                        class="body-1 text-white"
                        style="display: block; max-width: 100%;"
                      >
                        50 Points
                      </p>
                    </div>
                    <div
                      class="flex flex-col px-5 py-4"
                    >
                      <p
                        class="body-1 body-1 font-bold my-2"
                        style="display: block; max-width: 100%;"
                      >
                        Chronic Condition/Case Management Engagement (2 Consults)
                      </p>
                      <p
                        class="body-1"
                        style="display: block; max-width: 100%;"
                      >
                        After completing second consult | 1 per quarter | 100 points max per year
                      </p>
                    </div>
                  </section>
                </div>
                <div
                  class="card-button cursor-pointer mb-4"
                >
                  <section
                    class="flex flex-row"
                  >
                    <div
                      class="flex flex-col p-4 h-auto w-[72px] items-center text-center justify-center bg-primary-focus rounded-l-[8px]"
                    >
                      <p
                        class="body-1 text-white"
                        style="display: block; max-width: 100%;"
                      >
                        100 Points
                      </p>
                    </div>
                    <div
                      class="flex flex-col px-5 py-4"
                    >
                      <p
                        class="body-1 body-1 font-bold my-2"
                        style="display: block; max-width: 100%;"
                      >
                        Cancer Screenings (Mammogram, Colorectal, Cervical or Prostate)
                      </p>
                      <p
                        class="body-1"
                        style="display: block; max-width: 100%;"
                      >
                        Only 1 screening per year
                      </p>
                    </div>
                  </section>
                </div>
                <div
                  class="card-button cursor-pointer mb-4"
                >
                  <section
                    class="flex flex-row"
                  >
                    <div
                      class="flex flex-col p-4 h-auto w-[72px] items-center text-center justify-center bg-primary-focus rounded-l-[8px]"
                    >
                      <p
                        class="body-1 text-white"
                        style="display: block; max-width: 100%;"
                      >
                        50 Points
                      </p>
                    </div>
                    <div
                      class="flex flex-col px-5 py-4"
                    >
                      <p
                        class="body-1 body-1 font-bold my-2"
                        style="display: block; max-width: 100%;"
                      >
                        Diabetes Management (A1C, Urine Test and Kidney Function)
                      </p>
                      <p
                        class="body-1"
                        style="display: block; max-width: 100%;"
                      >
                        1 test per year | 150 points max per year
                      </p>
                    </div>
                  </section>
                </div>
                <div
                  class="card-button cursor-pointer mb-4"
                >
                  <section
                    class="flex flex-row"
                  >
                    <div
                      class="flex flex-col p-4 h-auto w-[72px] items-center text-center justify-center bg-primary-focus rounded-l-[8px]"
                    >
                      <p
                        class="body-1 text-white"
                        style="display: block; max-width: 100%;"
                      >
                        100 Points
                      </p>
                    </div>
                    <div
                      class="flex flex-col px-5 py-4"
                    >
                      <p
                        class="body-1 body-1 font-bold my-2"
                        style="display: block; max-width: 100%;"
                      >
                        Healthy Maternity Completion
                      </p>
                      <p
                        class="body-1"
                        style="display: block; max-width: 100%;"
                      >
                        1 per year max
                      </p>
                    </div>
                  </section>
                </div>
                <span
                  style="display: block; width: 1px; min-width: 1px; height: 16px; min-height: 16px;"
                />
                <a
                  aria-label="View All Ways to Earn Points"
                  style="display: block;"
                >
                  <button
                    class="flex flex-row link-container !flex"
                    style="max-width: max-content; height: auto; display: block;"
                    tabindex="-1"
                    type="button"
                  >
                    <p
                      class="link undefined"
                    >
                      View All Ways to Earn Points
                    </p>
                    <p
                      class="ml-1"
                    >
                      <img
                        alt=""
                        data-nimg="1"
                        decoding="async"
                        height="40"
                        loading="lazy"
                        src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                        srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                        style="color: transparent;"
                        width="40"
                      />
                    </p>
                  </button>
                </a>
              </div>
            </div>
          </div>
          <div
            class="flex flex-col flex-grow basis-6/12 items-stretch"
          >
            <div
              class="flex flex-col"
            >
              <div
                class="card-main large-section flex flex-row items-start app-body "
              >
                <div
                  class="flex flex-col"
                >
                  <div
                    class="flex flex-col"
                  >
                    <h2
                      class="title-2"
                    >
                      How to Earn Reward Points
                    </h2>
                    <span
                      style="display: block; width: 1px; min-width: 1px; height: 16px; min-height: 16px;"
                    />
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              How do I complete the PHA?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                        <div
                          class=""
                          style="width: 100%; height: 1px; background-color: rgb(204, 204, 204);"
                        />
                      </div>
                    </div>
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              When can I start completing the reward activities?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                        <div
                          class=""
                          style="width: 100%; height: 1px; background-color: rgb(204, 204, 204);"
                        />
                      </div>
                    </div>
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              How do I submit my daily steps?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                        <div
                          class=""
                          style="width: 100%; height: 1px; background-color: rgb(204, 204, 204);"
                        />
                      </div>
                    </div>
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              I have health issues that make completing some of the eligible wellness activities difficult. Are there any other ways I can earn points?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                      </div>
                    </div>
                    <span
                      style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
                    />
                  </div>
                </div>
              </div>
              <div
                class="card-main large-section flex flex-row items-start app-body "
              >
                <div
                  class="flex flex-col"
                >
                  <div
                    class="flex flex-col"
                  >
                    <h2
                      class="title-2"
                    >
                      About Points
                    </h2>
                    <span
                      style="display: block; width: 1px; min-width: 1px; height: 16px; min-height: 16px;"
                    />
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              How do I redeem my points?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                        <div
                          class=""
                          style="width: 100%; height: 1px; background-color: rgb(204, 204, 204);"
                        />
                      </div>
                    </div>
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              How much can my family earn?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                        <div
                          class=""
                          style="width: 100%; height: 1px; background-color: rgb(204, 204, 204);"
                        />
                      </div>
                    </div>
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              How can I check the status of my Visa Gift Card?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                        <div
                          class=""
                          style="width: 100%; height: 1px; background-color: rgb(204, 204, 204);"
                        />
                      </div>
                    </div>
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              Where can I view the points Iâ€™ve earned?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                      </div>
                    </div>
                    <span
                      style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
                    />
                  </div>
                </div>
              </div>
              <div
                class="card-main large-section flex flex-row items-start app-body "
              >
                <div
                  class="flex flex-col"
                >
                  <div
                    class="flex flex-col"
                  >
                    <h2
                      class="title-2"
                    >
                      About The Rewards Programs
                    </h2>
                    <span
                      style="display: block; width: 1px; min-width: 1px; height: 16px; min-height: 16px;"
                    />
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              When does each quarter end?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                        <div
                          class=""
                          style="width: 100%; height: 1px; background-color: rgb(204, 204, 204);"
                        />
                      </div>
                    </div>
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              What happens to my points if I donâ€™t earn 100 points by the end of the quarter?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                        <div
                          class=""
                          style="width: 100%; height: 1px; background-color: rgb(204, 204, 204);"
                        />
                      </div>
                    </div>
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              Youâ€™re missing my activity, or my points donâ€™t match what I think they should be. How do I get it corrected?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                        <div
                          class=""
                          style="width: 100%; height: 1px; background-color: rgb(204, 204, 204);"
                        />
                      </div>
                    </div>
                    <div
                      class="flex flex-col"
                    >
                      <div
                        class="flex flex-col items-stretch"
                      >
                        <div
                          class="flex flex-col card-main-elevated p-1 cursor-pointer px-2 py-2"
                        >
                          <div
                            class="flex flex-row m-1 accordion-head"
                          >
                            <p
                              class="mr-2 flex-grow"
                            >
                              If I accidentally delete the email with the digital gift card, can I get a new email/gift card sent to me?
                            </p>
                            <img
                              alt=""
                              class="pl-2 w-6"
                              data-nimg="1"
                              decoding="async"
                              height="40"
                              loading="lazy"
                              src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                              srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                              style="color: transparent;"
                              width="40"
                            />
                          </div>
                          <div
                            class="mr-2 flex-grow"
                          />
                        </div>
                      </div>
                    </div>
                    <span
                      style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </section>
      </div>
    </div>
  </div>
</body>
`;
